### DISCUSSION TITLE
Colored buttons with StavaasEVG‘s PlayStation Controller

### DISCUSSION THREAD

**Discussion by Gustour** [fayaz12g/totk-aar#39] [Ideas]
If I select StavaasEVG as controller, button are always white even if I choose colored. 
Not big issue but it will be perfect after that fix
Thanks, great 4.1.0 update.

**Comment by fayaz12g** [comment]
I've converted this into an idea. The StavaasEVG controller mods only support white buttons, so to add colored buttons would require me to either create those versions myself, or have the tool dynamically create controller mods. I've made this a stretch goal, but for now I've removed the option to customize layout and button color on the colored Dualsense
